src/java.base/share/classes/java/security/PEM.java

```diff
@@ -63,11 +81,11 @@
  * @since 25
  */
 @PreviewFeature(feature = PreviewFeature.Feature.PEM_API)
-public record PEMRecord(String type, String content, byte[] leadingData)
+public record PEM(String type, String content, byte[] leadingData)
     implements DEREncodable {
 
     /**
-     * Creates a {@code PEMRecord} instance with the given parameters.
+     * Creates a {@code PEM} instance with the given parameters.
```

Comment:
s/the given/the specified/

In the other ctor, you say what the parameters are, so you should do the same here.